haystack/components/converters/pdfminer.py

```diff
@@ -120,7 +120,7 @@ def _converter(self, extractor) -> Document:
         # Add a page delimiter
         concat = "\f".join(pages)
```

Comment:
just a nit - I think 'concat' as a verb is not very informal
```suggestion
        delimited_pages = "\f".join(pages)
```